### PR TITLE
DNS records changes

### PR DESCRIPTION
**Pull request by DaMandal0rian**
- add continuim.cc zone
- add subdomains for subspace.network
- renaming resources to follow convention

### CODE REVIEW

File: resources/dns/continuim-records.tf [hunk 1]
```diff
@@ -0,0 +1,17 @@
+resource "cloudflare_record" "main" {
```
Comment (vedhavyas) [inline resources/dns/continuim-records.tf]: maybe replace main with contunuim_cc ?

File: resources/dns/records.tf [hunk 2]
```diff
@@ -75,7 +71,40 @@ resource "cloudflare_record" "uptime" {
   ttl     = 1
   type    = "A"
   value   = "35.89.37.220"
-  zone_id = data.cloudflare_zone.cloudflare_zone.id
+  zone_id = data.cloudflare_zone.subspace_network.id
+}
+
+// subspace Grafana monitor
+resource "cloudflare_record" "grafana" {
```
Comment (vedhavyas) [inline resources/dns/records.tf:78]: If we have grafana exposed, do we have the auth setup on grafana ?

**Review by vedhavyas — COMMENTED** [review]
Make sense. Do we have auth enabled for grafana and other services we are exposing here ?

File: resources/dns/records.tf (hunk 2 above)
Comment (DaMandal0rian) [reply resources/dns/records.tf:78]: https://grafana.subspace.network/

**Review by vedhavyas — APPROVED** [review]

**Pull request merged**